src/Astreiko.Homework4/Program.cs

```diff
@@ -143,60 +138,61 @@ private static void EditEvent()
 
                     Console.Write("You want continious? (y - default or n)");
 
-                    if (Console.ReadLine().Trim() == "n") { checkEdit = false; }
-                };
+                    if (Console.ReadLine()?.Trim() == "n") { checkEdit = false; }
+                }
             }
         }
 
         /// <summary>
-        /// Прверка существования введенного идентификатора события
+        /// Checking for the existence of an entered event identifier (Проверка существования введенного идентификатора события)
         /// </summary>
         /// <returns></returns>
-        private static (bool BoolResult, int NumberRow) CheckEvent(string searchId)
+        private static int CheckEvent(string searchId)
         {
-            var vResB = false;
-            var vResI = 0;
+            var vRes = -1;
 
             for (int i = 0; i < CurrentListEvents.Count; i++)
             {
-                if (CurrentListEvents[i].Id == searchId) { vResB = true; vResI = i; };
+                if (CurrentListEvents[i].Id == searchId) { vRes = i; }
             }
 
-            return (vResB, vResI);
+            return vRes;
         }
 
         /// <summary>
-        /// Создание события
+        /// Create event
         /// </summary>
         private static void CreateEvent()
         {
             Console.Write("Enter description : ");
-            var inputDesc = Console.ReadLine().Trim();
+            var inputDesc = Console.ReadLine()?.Trim();
+
+            var inputStartDate = GetDate("Enter start date: ");
 
-            var inputDate = CheckDate();
+            var inputFinishDate = GetDate("Enter finish date: ");
 
             Console.Write("Enter status : ");
-            var inputStatus = Console.ReadLine().Trim();
+            var inputStatus = Console.ReadLine()?.Trim();
 
-            AddEventToList(inputDesc, inputDate.StartDate, inputDate.EndDate, inputStatus);
+            AddEventToList(inputDesc, inputStartDate, inputFinishDate, inputStatus);
         }
 
         /// <summary>
-        /// Получение введенных дат
+        /// Get entered date
         /// </summary>
         /// <returns></returns>
-        private static (DateTime StartDate, DateTime EndDate) CheckDate()
+        private static DateTime GetDate(string textToConsole)
         {
             var check = true;
 
-            var inputStartDate = DateTime.Now;
+            var inputDate = DateTime.Now;
 
             while (check)
             {
                 try
                 {
-                    Console.Write("Enter start date : ");
-                    inputStartDate = DateTime.Parse(Console.ReadLine());
+                    Console.Write(textToConsole);
+                    inputDate = DateTime.Parse(Console.ReadLine());
```

Comment:
you can return just from here, without initializing and checking additional variables